test/typescript/rules/custom.spec.ts

```diff
@@ -14,23 +63,26 @@ describe('TypeScript Definitions', () => {
 
             expect(check(10)).toEqual(true);
             expect(checker).toHaveBeenCalledTimes(1);
-            expect(checker).toHaveBeenCalledWith(10, schema, 'null', null, expect.any(Object));
+            expect(checker).toHaveBeenCalledWith(10, [], schema, 'null', null, expect.any(Object));
         });
 
         it('should call custom checker', () => {
-            const checker = jest.fn(() => true);
+            const checker = jest.fn((v) => v);
             const schema = { weight: { type: 'custom', a: 5, check: checker } };
             const check = v.compile(schema);
 
             expect(check({ weight: 10 })).toEqual(true);
             expect(checker).toHaveBeenCalledTimes(1);
-            expect(checker).toHaveBeenCalledWith(10, schema.weight, 'weight', { weight: 10 }, expect.any(Object));
+            expect(checker).toHaveBeenCalledWith(10, [], schema.weight, 'weight', { weight: 10 }, expect.any(Object));
         });
 
         it('should handle returned errors', () => {
-            const checker = jest.fn(function(value, schema, field) {
-                return [{ type: 'myError', field, expected: 3, actual: 4 }];
-            });
+            const fn: CheckerFunction<string> = function (value, errors, schema, field) {
+                errors.push({ type: 'myError', field, expected: 3, actual: 4 });
+                return value
+            }
+
+            const checker = jest.fn(fn as any);
```

Comment:
I don't know why it makes TypeError if I remove `as any`
I think it's problem from Jest Typing. because types are OK without using `jest.fn`